### PR TITLE
[Feat] pretty printing with custom inspect

### PR DESCRIPTION
**Pull request by DominusKelvin**
Currently the pretty printing of machines when using `utils.inspect` or `console.log` don't work mostly because of the change to the API of creating a custom inspect. This PR fixes the affect `inspect` implementation.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -66,15 +66,14 @@ module.exports.version = RELEASE_VERSION;//« for backwards compatibility
...
- * .inspect()
+ * util.inspect.custom
  *
  * When the Machine constructor is inspected (e.g. `util.inspect()` / `console.log()`),
  * pretty print the current version of node-machine, with license information and a link
  * to the documentation.
  *
- * @returns {String}
  */
-module.exports.inspect = function () {
```
Comment (mikermcneil) [inline index.js:77]: `module.exports[Symbol.for('nodejs.util.inspect.custom')] = `

File: index.js [hunk 2]
```diff
@@ -66,15 +66,14 @@ module.exports.version = RELEASE_VERSION;//« for backwards compatibility
 
 
 /**
- * .inspect()
+ * util.inspect.custom
```
Comment (mikermcneil) [inline index.js:69]:
```
 * [Symbol.for('nodejs.util.inspect.custom')]
 * (formerly known as ".inspect()")
 * https://github.com/node-machine/machine/pull/50/files
```

File: lib/pack.js [hunk 3]
```diff
@@ -408,41 +408,33 @@ module.exports = function pack(optionsOrDir){
 
 
   /**
-   * .inspect()
+   * util.inspect.custom
```
Comment (mikermcneil) [inline lib/pack.js:411]: same as above (i know it's a lot but I know how forgetful I am)

File: lib/private/help-build-machine.js [hunk 4]
```diff
@@ -1723,224 +1723,216 @@ module.exports = function helpBuildMachine(opts, btOmen) {
 
 
   /**
-   * .inspect()
+   * util.inspect.custom
```
Comment (mikermcneil) [inline lib/private/help-build-machine.js:1726]: same as above (i know it's a lot but I know how forgetful I am)

**Review by mikermcneil — CHANGES_REQUESTED** [review]
Other than what I mentioned, just wanna minimize the redlines and stick with defineProperty

**Review by eashaw — APPROVED** [review]

**Pull request merged**